VEP 99.2 foss 2019b

afraz-ahmed opened the pull request:
Needs #107 

For INC1019168

VEP-99.2-foss-2019b-Perl-5.30.0.eb
* [x] Assigned to reviewer
* [ ] EL7-cascadelake
* [ ] EL7-haswell
* [ ] EL7-sandybridge
* [ ] Ubuntu16 VM

branfosj commented:
We'll need to alter this from the upstream version - one the people who uses VEP always wants an a specific extra plugin to work and that involves an addition: https://gitlab.bham.ac.uk/res-comp-team/bear-software/2019a/easybuild-easyconfigs/-/blob/master/easybuild/easyconfigs/v/VEP/VEP-96.0-foss-2019a-Perl-5.28.1.eb

afraz-ahmed commented:
> We'll need to alter this from the upstream version - one the people who uses VEP always wants an a specific extra plugin to work and that involves an addition: https://gitlab.bham.ac.uk/res-comp-team/bear-software/2019a/easybuild-easyconfigs/-/blob/master/easybuild/easyconfigs/v/VEP/VEP-96.0-foss-2019a-Perl-5.28.1.eb

Hi, Simon. I have added the extra files and updated all dependencies. Thanks.

branfosj commented on easybuild/easyconfigs/b/Bio-DB-BigFile-jksrc/Bio-DB-BigFile-jksrc-v395-foss-2019b.eb:
```diff
@@ -0,0 +1,43 @@
+name = 'Bio-DB-BigFile-jksrc'
+version = 'v395'
+
+homepage = "http://hgdownload.cse.ucsc.edu/admin/"
```
```suggestion
homepage = "https://hgdownload.cse.ucsc.edu/admin/"
```

branfosj commented on easybuild/easyconfigs/b/Bio-DB-BigFile-jksrc/Bio-DB-BigFile-jksrc-v395-foss-2019b.eb:
```diff
@@ -0,0 +1,43 @@
+toolchain = {'name': 'foss', 'version': '2019b'}
+toolchainopts = {'pic': True}
+
+source_urls = ['http://hgdownload.cse.ucsc.edu/admin/']
```
```suggestion
source_urls = ['https://hgdownload.cse.ucsc.edu/admin/']
```

branfosj commented on easybuild/easyconfigs/l/LibUUID/LibUUID-1.0.3-GCCcore-8.3.0.eb:
```diff
@@ -0,0 +1,40 @@
+name = 'LibUUID'
+version = '1.0.3'
+
+homepage = 'http://sourceforge.net/projects/libuuid/'
```
```suggestion
homepage = 'https://sourceforge.net/projects/libuuid/'
```

branfosj commented on easybuild/easyconfigs/b/Bio-DB-BigFile-jksrc/Bio-DB-BigFile-jksrc-v395-foss-2019b.eb:
```diff
@@ -0,0 +1,42 @@
+]
+
+dependencies = [
+    ('BioPerl', '1.7.2', '-Perl-5.30.0'),
```
```suggestion
    ('BioPerl', '1.7.2'),
```

branfosj commented on easybuild/easyconfigs/b/Bio-DB-BigFile/Bio-DB-BigFile-1.07-foss-2019b-Perl-5.30.0.eb:
```diff
@@ -0,0 +1,33 @@
+
+dependencies = [
+    ('Perl', '5.30.0'),
+    ('BioPerl', '1.7.2', versionsuffix),
```
```suggestion
    ('BioPerl', '1.7.2'),
```

branfosj commented on easybuild/easyconfigs/v/VEP/VEP-99.2-foss-2019b-Perl-5.30.0.eb:
```diff
@@ -0,0 +1,68 @@
+dependencies = [
+    ('Perl', '5.30.0'),
+    ('DBD-mysql', '4.050', versionsuffix),
+    ('BioPerl', '1.7.2', versionsuffix),
```
```suggestion
    ('BioPerl', '1.7.2'),
```

edmondac commented:
Build failed with:
```
Can't locate ExtUtils/CppGuess.pm in @INC (you may need to install the ExtUtils::CppGuess module 
```
